Loop/Views/DosingStrategySelectionView.swift

```diff
@@ -45,27 +47,52 @@ public struct DosingStrategySelectionView: View {
                             self.internalDosingStrategy = strategy // Hack to force update. :(
                         }
                     }
-                )
+                ),
+                trailingView: strategy.isAutomaticBolus ? linearRampVolusApplicationFactorSection : nil
             )
             .padding(.vertical, 4)
         }
     }
 }
 
+extension DosingStrategySelectionView {
+    var linearRampVolusApplicationFactorSection: AnyView {
```

Comment:
Would prefer that this go in an entirely new section entitled "Algorithm Experiments", separate from bolus/basal dosing strategy, and be a single line with the title and an "on/off" *label*, and a disclosure arrow.  Tapping on the line would take you a full page description of the option, and the ability to enable/disable it.

I understand that it's convenient to make it part of the dosing strategy, since it doesn't apply to Temp Basal, but this should be kept separate in the UI for now, as it's an experimental option.